From 97cd90b33cc988dedeac6c4c94a1e40fd5c58488 Mon Sep 17 00:00:00 2001
From: Lionel Xu <Lionel.Xu@freescale.com>
Date: Fri, 2 Mar 2012 12:57:51 +0800
Subject: ENGR00170526-1 MX6Q_Sabreauto: remove audio platform data rst_gpio.

Remove audio platform data rst_gpio which is not longer required now.

Signed-off-by: Lionel Xu <Lionel.Xu@freescale.com>
---
 arch/arm/mach-mx6/board-mx6q_sabreauto.c |    1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm/mach-mx6/board-mx6q_sabreauto.c b/arch/arm/mach-mx6/board-mx6q_sabreauto.c
index b45bd37..d7fc460 100644
--- a/arch/arm/mach-mx6/board-mx6q_sabreauto.c
+++ b/arch/arm/mach-mx6/board-mx6q_sabreauto.c
@@ -1237,7 +1237,6 @@ static const struct pm_platform_data mx6q_sabreauto_pm_data __initconst = {
 
 static struct mxc_audio_platform_data sab_audio_data = {
 	.sysclk = 24576000,
-	.rst_gpio = MX6Q_SABREAUTO_PER_RST,
 	.codec_name = "cs42888.1-0048",
 };
 
-- 
1.7.9.5

